go/pserver/service_test.go

```diff
@@ -30,7 +30,7 @@ const (
 
 func TestServiceFull(t *testing.T) {
 	var cp pserver.Checkpoint
-	s, err := pserver.NewService(0, 1, "", nil, cp)
+	s, err := pserver.NewService(0, time.Hour, "", nil, cp)
```

Comment:
Good point! checkpoint requires etcd in test. We can add testing checkpoint in future PR.